lib/libutee/include/rng_pta_client.h

```diff
@@ -1,45 +1,14 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /*
- * Copyright (C) 2018, Linaro Limited
+ * Copyright (C) 2018-2022, Linaro Limited
+ *
+ * This header file will sooner or later be deprecated, unless endlessly
+ * maintained for backward compatibility.
```

Comment:
With this PR, it is deprecated already, and may be *removed* sooner or later. So I suggest:
"This header file is deprecated and kept for backward compatibility."